### PR TITLE
Remove dependency on Knative

### PR DESCRIPTION
**Pull request by juliababkina**
Solution for the issue pivotal#69

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -23,6 +23,7 @@ require (
 	github.com/mattbaird/jsonpatch v0.0.0-20171005235357-81af80346b1a // indirect
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.1 // indirect
+	github.com/nyarly/spies v0.0.0-20200413230442-112961b2b018 // indirect
```
Comment (qelo) [inline go.mod:20]: Is this intended? Why does removing Knative add a new dep?

File: go.mod [hunk 2]
```diff
@@ -23,6 +23,7 @@ require (
 	github.com/mattbaird/jsonpatch v0.0.0-20171005235357-81af80346b1a // indirect
```
Comment (qelo) [inline go.mod:17]: Why no deps are removed, shoudn't at least knative deps get removed?

File: pkg/model/replica_entity.go [hunk 3]
```diff
@@ -57,50 +50,26 @@ type replicaEntity struct {
...
-
-	endpoints.Subsets[0].Addresses = append(endpoints.Subsets[0].Addresses, re.endpointAddress)
-
-	updatedEndpoints, err := re.kubernetesClient.CoreV1().Endpoints("skenario").Update(endpoints)
-	if err != nil {
-		panic(err.Error())
-	}
-	err = re.endpointsInformer.Informer().GetIndexer().Update(updatedEndpoints)
+	now := re.env.CurrentMovementTime().UnixNano()
+	err := re.env.Plugin().Event(now, proto.EventType_CREATE, &skplug.Pod{
```
Comment (qelo) [inline pkg/model/replica_entity.go:54]: Is this related to removing Knative? Looks like some new code.

File: go.mod (hunk 1 above)
Comment (juliababkina) [reply go.mod:20]: No, it left after debug. I should remove it.

File: go.mod (hunk 2 above)
Comment (juliababkina) [reply go.mod:17]: You're right. I should have double checked go.mod file. I was quite sure that they would be removed automatically from go.mod. I'll remove them.

File: pkg/model/replica_entity.go (hunk 3 above)
Comment (juliababkina) [reply pkg/model/replica_entity.go:54]: I just moved this code.

File: go.mod (hunk 1 above)
Comment (juliababkina) [reply go.mod:20]: Fixed it, see commit 577c562e5c2def8fb5d9c8a04b906f1ed59ba857.

File: go.mod (hunk 2 above)
Comment (juliababkina) [reply go.mod:17]: Fixed them, see commit 577c562e5c2def8fb5d9c8a04b906f1ed59ba857.

**Review by josephburnett — APPROVED** [review]
/lgtm

**Review by qelo — APPROVED** [review]
